curse.go

```diff
@@ -40,7 +40,7 @@ func New() (*Cursor, error) {
 	c := &Cursor{}
 	c.Position.X, c.StartingPosition.X = col, col
 	c.Position.Y, c.StartingPosition.Y = line, line
-	c.terminal, err = term.New()
+	c.terminal, err = term.Open("/bin/stty")
```

Comment:
I have an older Go program that I resurrected that uses `curse` and it gets an error here:
`open /bin/stty: operation not permitted`
I've tried it as root too just to be sure.